Automattic/jetpack
# Change/jetpack contact form icon

Wee change: replace Jetpack Contact Form icon

Fixes #26865 

#### Changes proposed in this Pull Request:
- replace Jetpack Contact Form icon with the one provided


Before:
<img width="25" alt="image" src="https://user-images.githubusercontent.com/157240/197284602-98adaced-10ec-4130-a7d3-009e9fa9be05.png">


After:
<img width="29" alt="image" src="https://user-images.githubusercontent.com/157240/197284529-cb13817e-f8b0-44ed-95e9-de3868c11e24.png">



#### Other information:


#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
Apply patch and insert a `/form`, you should see the new icon:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/157240/197284417-fa89f371-ee08-4663-af8f-023dce46770f.png">
